README.md

```diff
@@ -0,0 +1,26 @@
+#Rebuy-Forms-Rating
+
+##What is this?
+Rebuy-Forms-Rating is a Xamarin.Forms ActionSheet for rating the app in the appstores (Apple AppStore, Google PlayStore).
+
+We are very eager about your feedback, so do not hesitate to create an issue or feel free to improve our code via a contribution.
+
+###Features 
+ - Fully Xamarin.Forms compatible.
```

Comment:
At the moment I use the 1.5.0.6447 version.

Took that line from the rb.barcode.library readme file